.github/dependabot.yml

```
@@ -0,0 +1,25 @@
+
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "monthly"
+    cooldown:
+      default-days: 7
+    groups:
+      github-actions:
+        patterns:
+          - "*"
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)